network-api/networkapi/wagtailpages/templates/wagtailpages/blog_page.html

```diff
@@ -55,10 +55,9 @@ <h1 class="tw-h1-heading">{{ page.title }}</h1>
                 <div class="streamfield-content">
                     <div class="newsletter-signup-module"
                          data-module-type="blog-body"
-                         data-signup-id="value_coming_from_cms"
-                         data-cta-header="Reclaim the Internet"
-                         data-cta-description="Mozilla is fighting for a free and open web for everyone. But there’s only one way we can do it - together. Sign up now to join the Mozilla community."
-                         data-newsletter="{{ value_coming_from_cms }}"
+                         data-signup-id="{{ localized_signup.id }}"
+                         data-cta-header=""
```

Comment:
header and description value will eventually be hooked up with the to-be-built snippet